src/Mutator/Util/MutatorsGenerator.php

```diff
@@ -99,7 +99,7 @@ private function registerFromClass(string $mutator, $setting)
                 $this->mutatorList[$mutator] = [];
             }
         } elseif (!array_key_exists($mutator, $this->mutatorList) || !$this->mutatorList[$mutator]) {
-            $this->mutatorList[$mutator] = (array) $setting;
+            $this->mutatorList[$mutator] = $setting;
```

Comment:
just curious, what was the issue reported by PHPStan here?

I removed this `(array)` cast and then reverted during `@cast` mutators developing, but don't remember why

Reply:
It reported a useless to array cast, but it looks like i messed up some other things, so i'll go ahead and fix that first.

Reply:
As it turns out, this wasn't a useless cast, because the input can be a `\stdClass` as well.